docs/sources/datasources/google-cloud-monitoring/_index.md

```diff
@@ -182,6 +182,10 @@ Example Result: `gce_instance - compute.googleapis.com/instance/cpu/usage_time`
 Click on a time series in the panel to see a context menu with a link to View in Metrics Explorer in Google Cloud Console. Clicking that link opens the Metrics Explorer in the Google Cloud Console and runs the query from the Grafana panel there.
 The link navigates the user first to the Google Account Chooser and after successfully selecting an account, the user is redirected to the Metrics Explorer. The provided link is valid for any account, but it only displays the query if your account has access to the GCP project specified in the query.
 
+#### Automatic unit detection
+
+Grafana issues one query to the Cloud Monitoring API per query editor row, and each API response includes a unit. Grafana will attempt to convert the returned unit into a unit that is understood by the Grafana graph panel. If conversion was successful, the unit will be displayed on the Y-axis on the panel. In case the query editor rows returned different units, the unit from the last query editor row will be used in the graph panel.
```

Comment:
```suggestion
Grafana issues one query to the Cloud Monitoring API per query editor row, and each API response includes a unit. Grafana will attempt to convert the returned unit into a unit that is understood by the Grafana graph panel. If conversion was successful, the unit will be displayed on the Y-axis on the panel. If the query editor rows returned different units, then the unit from the last query editor row is used in the graph panel.
```

Reply:
@sunker, this section looks like it is written specifically for the Graph panel, which will be replaced eventually by the Time series panel. It might be a good time to adjust the verbiage.

Reply:
good catch!